lms/templates/navigation.html

```diff
@@ -85,7 +85,8 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
               </a>
             </li>
             <li class="tab-nav-item">
-              <a class="${'active ' if reverse('program_listing_view') == request.path else ''}tab-nav-link" href="${reverse('program_listing_view')}">
+              ## TODO: Update to user actual program ids
+              <a class="${'active ' if (reverse('program_listing_view') == request.path) or (reverse('program_details_view', args=[1]) == request.path) else ''}tab-nav-link" href="${reverse('program_listing_view')}">
```

Comment:
Maybe you could check `if reverse('program_listing_view') in request.path`? Given the structure of the URLs, that should net you anything under `/programs/`.